src/parser.ts

```diff
@@ -713,17 +713,17 @@ export class Parser extends DiagnosticEmitter {
           if (tn.skip(Token.COLON)) {
             isSignature = true;
             tn.discard(state);
-            let t = this.parseType(tn, false);
-            if (!t) return null;
-            if (t.kind != NodeKind.NAMEDTYPE) {
+            let ty = this.parseType(tn, false);
```

Comment:
Can we name these `type` or leave it `t`? Associating something math with `ty` for some reason :)

Reply:
sure